core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -283,6 +341,42 @@ oppia.controller('ExplorationEditor', [
       });
     };
 
+    $scope.showWelcomeExplorationModal = function() {
+      $scope.welcomeModalIsOpening = true;
+      var startTutorial = $scope.startTutorial;
+      var modalInstance = $modal.open({
+        templateUrl: 'modals/welcomeExploration',
+        backdrop: true,
+        controller: [
+          '$scope', '$modalInstance', function($scope, $modalInstance) {
+            $scope.beginTutorial = $modalInstance.close;
+
+            $scope.cancel = function() {
+              $modalInstance.dismiss('cancel');
+            };
+          }],
+        windowClass: 'oppia-welcome-modal'
+      });
+
+      modalInstance.result.then(function() {
+      // Called after $modal.close()
+      startTutorial();
+    }, function() {
+      // Called after $modalInstance.dismiss
+      stateEditorTutorialFirstTimeService.markTutorialFinished();
```

Comment:
Never mind. It only shows the first time because that whole thing is run only once.